### PR TITLE
remove BUILD spammy output

### PR DESCRIPTION
**Pull request by mikedanese**
leftover from debugging

### CODE REVIEW

File: test/e2e/BUILD [hunk 1]
```diff
@@ -244,7 +244,7 @@ genrule(
     testonly = 1,
     srcs = [":go_default_test"],
     outs = ["e2e.test"],
-    cmd = "set -x; srcs=($(SRCS)); cp $$(dirname $${srcs[0]})/go_default_test $@;",
```
Comment (ixdy) [inline test/e2e/BUILD]: while you're here, can you also add `output_to_bindir = 1` to this rule? otherwise `e2e.test` ends up in `bazel-genfiles/`.

Comment (mikedanese) [inline test/e2e/BUILD]: DOne.

**Comment by ixdy** [comment]
@k8s-bot unit test this

**Comment by mikedanese** [comment]
This test is very flaky

**Pull request merged**